flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/main/java/org/apache/flink/cdc/connectors/paimon/sink/PaimonMetadataApplier.java

```diff
@@ -160,15 +156,24 @@ private void applyCreateTable(CreateTableEvent event) throws SchemaEvolveExcepti
                     .forEach(
                             (column) ->
                                     builder.column(
-                                            column.getName(),
+                                            column.getName().toLowerCase(),
```

Comment:
Hi yanquan, copy that. But both `filesystem` and `hive` will create table through `applyCreateTable`, so I think it makes sense?

Reply:
For users who use the `filesystem` as metastore, changing to lowercase when creating tables may not affect their usage. But it may be unexpected and they may have to adapt to our changes instead.

Reply:
Well, I've reviewed the codes, we can pass the parameter of `metastore` (`filesystem` and `hive`) to judge whether to use lowercase, but that should add `metastore` in schema, and `schema` is in common classes.
So I think maybe it's better to keep it unchanged?